test/commands/trigger_test.exs

```diff
@@ -1,13 +1,15 @@
 defmodule Cog.Test.Commands.TriggerTest do
-  use Cog.CommandCase, command_module: Cog.Commands.Trigger
+  use Cog.CommandCase, command_tag: :trigger
```

Comment:
@mpeck you can always add the `commands: :trigger` module tag to each file. If you wanted to run a single subcommand, you'd just execute that file directly.